internal/server/data/migrations.go

```diff
@@ -81,8 +81,9 @@ func migrations() []*migrator.Migration {
 		addUserPubicKeysTable(),
 		addUserSSHLoginName(),
 		makeIdxEmailsProvidersUnique(),
+		addDestinationCredentials(),
```

Comment:
minor: putting this below `deviceFlowAuthRequestsAddUserIDProviderID` will ensure the migrations will be run in tests, in the same order as they did in dev and prod.